### PR TITLE
Replace and split issue templates. Update contributing information

### PR DESCRIPTION
**Pull request by micolous**
Replaces the current issue template with new split templates, so that the questions are more relevant.

Rewrite the "contributing" document so that it doesn't duplicate as much information, and it is more up-to-date.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_card_felica.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Is the card similar to one we already support? (if you don't know, that's OK)
+
+Is there an official app for using the card?
+
+What are the system and service codes on the card?  (eg: System 0x3 services: 0x1, 0x2)
+
+With the data that you can read from the card, does it change after performing a transaction?
+
+
+### Extra required questions for ISO 7816
```
Comment (phcoder) [inline .github/ISSUE_TEMPLATE/new_card_felica.md]: This is not relevant to Felica

File: .github/ISSUE_TEMPLATE/new_card_vicinity.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] I have attempted to understand the card format on my own.
+- [ ] I have provided notes on what I've found so far.
+
+## Please answer all questions listed here:
+
+- Name of the card:
+- Name of the transit agency:
+- Transit agency's website:
+- Location(s):
+- Card media: MIFARE Classic
```
Comment (phcoder) [inline .github/ISSUE_TEMPLATE/new_card_vicinity.md]: Copy-paste?

File: .github/ISSUE_TEMPLATE/new_card_felica.md (hunk 1 above)
Comment (micolous) [reply .github/ISSUE_TEMPLATE/new_card_felica.md]: Fixed

File: .github/ISSUE_TEMPLATE/new_card_vicinity.md (hunk 2 above)
Comment (micolous) [reply .github/ISSUE_TEMPLATE/new_card_vicinity.md]: Fixed

**Comment by micolous** [comment]
I've updated this to fix some issues I found, after trying it out in a test repository.

**Comment by micolous** [comment]
Discussed via Hangouts. This doesn't change any code, so CI flakes don't matter. :)

**Pull request merged**